### ISSUE TITLE
Mashing up Multiple WebIDs and Pods

### ISSUE THREAD

**Issue by jeff-zucker** [solid/solidos#55]
Tim Berners-Lee @timbl Aug 26 14:17 said in gitter/solid/chat

> @joepio Thanks for the link to all the UI material and ideas.
I’d like us to design interfaces to our sold stuff, whatatever pods it may be on. I’d like to be able to by default look at stuff in four pods, and at each point, say looking at photo albums or photos, be able to turn on and off different pods from the merged view.
Like layers on a map.

He reiterated similar ideas in the SolidOS meeting on 2021-09-08 and also mentioned a related feature - the ability to use multiple identities simultaneously while keeping the identities hidden.  If I'm understanding correctly, combining this idea with the one above would mean I could be looking at a mashup of my photos from my work and play IDs without  anyone but me knowing that I am associated with both IDs.

My thoughts are that these kinds of things are most easily doable by authorizing an ID for multiple pods - if my workID Pod and my playID Pod both assign the homeID as authorized to read photos I'm all set after one login.   But it might also be worth considering the longer route for SolidOS -  let all WebIDs operate simultaneously.  If I try to get resource X with WebID Y and that doesn't work, the app tries WebID Z or offers a new login prompt.

**Comment by timbl** [comment]
Yes, you got the idea, thanks for elaborating.  I can using multiple IDs in different ways. 

1. One mode as you say just try all my credentials until you get me what I want. If you have several public IDs which are publicly known to be the same, then you can use this mode and systems will figure out they are the same person even if they didn't know before.

2. The other mode is more controlled, more deterministic, better for privacy.  Each time for the first time you have to authenticate to a different domain, thee system asks you which webid you want to use to log n, and it remembers forever in your pod's preferences.  So you can say "log into pensions.gov with my home ID, and the solid project with my work ID."  That way each pod server doesn't have to know that they belong to the same person, and you can have different preferences for world.  This is the way the browser used to work with TLS certs if you had more than one cert.

**Comment by timbl** [comment]
Note that this doesn't just apply to pods, it applies to accessing public data sources too like OpenStreepMap and Wikidata, and private data sources like feeds from data firms and data resources behind paywalls.

**Comment by jeff-zucker** [comment]
[The Open Culture Browser](https://jeff-zucker.github.io/linked-bookmarks/) is one example of mashing up data from public sources (wikidata + internet archive).   But it does not deal with logins and identities.

One way of experimenting with multiple IDs is in Data-Kitchen.  I believe it's possible to open multiple independent browsing contexts (each with its own webID, for example) and to pull to a central location from all the contexts.   From the user's point of view, everything could happen in one window/tab even if other windows/tabs might be operating headless in the background.

**Comment by jeff-zucker** [comment]
I might have several pods for each identity.  So I'd imagine something like this in the profile : 
```turtle
<#me>
  :identity 
    [
      foaf:nick "Work" ;
      solid:oidcIssuer <https://broker.pod.inrupt.com> ;
      :domain <#workPod1>, <#workPod2> .
    ],
    [
      foaf:nick "Play" ;
      solid:oidcIssuer <https://solidcommunity.net> ;
      :domain <#playPod1>, <#playPod2> .
    ] .
```
